src/plugins/data_table/data_table.js

```diff
@@ -243,6 +243,47 @@ var LocalDataTable = (function() {
       });
     },
 
+    _renderGrandTotalsRow: function() {
+      var hasGrandTotalsCell = false;
+      // if dataTable is available AND we have a totals row
+      if (this.dataTable && this.totalsRow) {
+        // If we don't have a footer rendered, render it
+        if (this.dataTable.find("tfoot").length < 1) {
+          this.dataTable.append("<tfoot><tr></tr></tfoot>");
+        }
+        // Iterate over the current columns config
+        var grandTotalsRowCellNodes = _.map(this.columnsConfig(), function(col) {
+          // If a column is visible, render its total
+          if (this.columnVisibility(col.id) || col.bulk) {
+            /***
+              TODO: Add some sort of warning if the user has a totals property BUT
+              doesn't have a isNontotalsColumn predicate test.
+            ***/
```

Comment:
do we need a warning?  why can't we allow a table where every single column has data?

Either way, please either add the warning or remove this TODO..